File: app/src/main/java/com/android/learn/mcontract/WechatContract.java
package com.android.learn.mcontract;

import com.android.learn.base.mmodel.BannerListData;
import com.android.learn.base.mmodel.FeedArticleListData;
import com.android.learn.base.mmodel.FeedArticleListData.FeedArticleData;
import com.android.learn.base.mmodel.WxArticle;

import java.util.List;

/**
 * Created by gaolei on 2018/6/18.
 */

public class WechatContract {

    public interface Presenter {

        void getWxArticle();

    }

    public interface View {

        void showWxArticle(List<WxArticle> list);


    }
}

File: app/src/main/java/com/android/learn/mcontract/WechatContract.kt
package com.android.learn.mcontract

import com.android.base.mmodel.WxArticle
import com.android.base.mview.BaseView

/**
 * Created by gaolei on 2018/6/18.
 */

class WechatContract {

    interface Presenter {

        fun getWxArticle()

    }

    interface View : BaseView {

        fun showWxArticle(list: List<WxArticle>)


    }
}
